.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -8,8 +8,126 @@ on:
         required: true
 
 jobs:
-  build:
+  build-binaries:
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - { os: windows-latest, r: 'release'}
+          - { os: macOS-latest, r: 'release'}
+          - { os: ubuntu-latest, r: 'release'}
+    environment: ci_build
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: ${{ matrix.config.r }}
+
+      - name: Install openssl MacOsX
+        if: matrix.config.os == 'macOS-latest'
+        run: |
+          brew install openssl
+          BREW_PREFIX=`brew --prefix`
+      - name: Install Ubuntu package dependencies
+        if: matrix.config.os == 'ubuntu-latest'
+        run: |
+            sudo apt-get -y update && \
+               sudo apt-get -y install --no-install-recommends libcurl4-openssl-dev curl libxml2-dev  libssl-dev && \
+               sudo locale-gen en_US.UTF-8 && \
+               sudo update-locale LC_ALL=en_US.UTF-8 && \
+               sudo apt-get -y clean && \
+               sudo apt-get -y autoremove
+      - name: Query dependencies
+        run: |
+          install.packages("RODBC")
+          install.packages("DBI")
+          install.packages("devtools", dependencies=TRUE)
+          devtools::install_deps()
+        shell: Rscript {0}
+        env: # Set individual GH PAT
+          GITHUB_PAT: ${{ secrets.PAT_FOR_DEVTOOLS }}
+      - name: Create output dir
+        run: |
+            out_put_dir="${{ runner.temp }}/bin/${{ matrix.config.os }}"
+            out_put_dir=$(echo "$out_put_dir" | sed  -e 's/\\/\//g') #Needed on Windows
+            mkdir -p "${out_put_dir}"
+            echo "output_dir=${out_put_dir}" >> $GITHUB_ENV
+        shell: bash
+      - name: Build
+        run: |
+          devtools::build(binary=TRUE, path='${{ env.output_dir }}')
+        shell: Rscript {0}
+        env: # Set individual GH PAT
+          GITHUB_PAT: ${{ secrets.PAT_FOR_DEVTOOLS }}
+      - name: Check result
```

Comment:
Should this have `ubuntu-latest` condition? Without it, it seems to be run for all distros.

Reply:
Hm, this workflow runs only on ubuntu-latest, check line 92